server/migrations/20170817134614-create-user.js

```diff
@@ -1,45 +1,50 @@
 'use strict';
 module.exports = {
-  up: function(queryInterface, Sequelize) {
-    return queryInterface.createTable('Users', {
+  up: (queryInterface, Sequelize) => 
```

Comment:
No point commenting out the Sequelize argument down the line, you should remove it if its not in use.